src/land/smap_ssm2ioda.py

```diff
@@ -85,6 +86,8 @@ def _read(self):
         lons = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['longitude'][:].ravel()
         errs = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['soil_moisture_error'][:].ravel()
         qflg = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['retrieval_qual_flag'][:].ravel()
+
+        deps = np.full_like(vals, 0.025)
```

Comment:
here's that same value,,, may be good to centralize its value to a constant somewhere

Reply:
Not sure what this means. @ClaraDraper-NOAA, could you please give a response for this comment.

Reply:
Can we set a variable called:

`assumed_obsevation_depth = 0.025   #observation assumed to be 2.5cm or 0.025m`

and then use this in the two places where it appears... makes it easier to possible make this be something which can be passed in via yaml or arguments

Reply:
If you want me to add comments like this: deps = np.full_like(vals, 0.025)  #observation assumed to be 0.025m, I can add it to my python code.  Do you really want @BenjaminRuston?

Reply:
but the number 0.025 appears in two separate places so it seems like a good one to consolidate so they point to a single variable, don't you think?

sure using depthBelowSoilSurface is fine, but would you agree this may be better than having 0.025 hard coded in two separate locations.... and yes long names because someone new may not immediately know what `deps` means

Reply:
Okay, you mean that I can define it as assumed_obsevation_depth = 0.025 #observation assumed to be 2.5cm or 0.025m in the beginning of this python code, and then put the constant value via deps = np.full_like(vals, assumed_obsevation_depth), right? @BenjaminRuston

Reply:
@YoulongXia-NOAA no worries. I think what @BenjaminRuston means by "in two separate locations..." is that the hard coded value 0.025 appears once in each of the two scripts (smap9km_ssm2ioda.py and smap_ssm2ioda.py). I think what is being suggested is to have a parameter that is defined in only one place and used by both of the scripts.

We should have a common (shared) file for situations like this where you define common parameters to be used in multiple scripts. I'm not sure if we have that yet for the converters. This would be a nice long term solution.

Perhaps for the short term it would be acceptable to define the parameter at the beginning of each script so that if it ever gets used in multiple places within a script you've got an easier way to update the value. Thinking about a shared file perhaps a good name for the parameter would be something like `assumedSoilDepth = 0.025`. Then when the shared file scheme gets implemented, you would simply delete the setting of the parameter at the beginning to the two scripts and import the shared file with the parameter setting.

Setting up the shared parameter file could be done in a subsequent PR after this one.

Does this sound like an acceptable approach?